apis/servicenetworking/v1beta1/connection_types.go

```diff
@@ -93,9 +88,11 @@ type ConnectionParameters struct {
 	// `{project}` is a project number, such as in `12345` that includes
 	// the VPC service consumer's VPC network. `{network}` is the name of
 	// the service consumer's VPC network.
+	// +optional
 	Network string `json:"network,omitempty"`
 
 	// NetworkRef references to a Network and retrieves its URI
+	// +optional
```

Comment:
Wasn't sure how to handle it because it is technically optional because a user may reference a `Network` type, but it will always be populated by the time we reconcile.

Reply:
I think we usually go for pointers in this case - probably best to do so here.